applications/StructuralMechanicsApplication/custom_utilities/shell_cross_section.hpp

```diff
@@ -414,36 +415,34 @@ class KRATOS_API(STRUCTURAL_MECHANICS_APPLICATION) ShellCrossSection : public Fl
 
     private:
 
-        double mThickness;
-        double mLocation;
+        int mPlyIndex;
         double mOrientationAngle;
         IntegrationPointCollection mIntegrationPoints;
         Properties::Pointer mpProperties;
 
     public:
 
         Ply()
-            : mThickness(0.0)
-            , mLocation(0.0)
-            , mOrientationAngle(0.0)
+            : mPlyIndex(0)

```

Comment:
You removed `mOrientationAngle(0.0)` when you replaced `mThickness` and `mLocation` by `mPlyIndex`. I suspect this is accidental since now the member variable mOrientationAngle is not initialized in the default constructor. This style of default initialization is a known source of bugs which is one reason why I think there is a strong consensus that one should prefer in class member initialization for defaults (http://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Rc-default). I would suggestion always switching to in class initialization whenever you have to update a constructor that uses this buggish style.

Reply:
I forgot to remove mOrientationAngle
Thx for spotting!